content/en/docs/ambient/install/helm-installation/index.md

```diff
@@ -114,8 +138,48 @@ After installing ambient mode with Helm, you can follow the [Deploy the sample a
 
 ## Uninstall
 
-You can uninstall Istio and its components by uninstalling the charts
-installed above.
+If you installed with the `ambient` wrapper chart above, you can uninstall Istio and its components by uninstalling that chart:
+
+1. Delete any Istio gateway chart installations (optional, if installed):
+
+    {{< text syntax=bash snip_id=delete_ingress >}}
+    $ helm delete istio-ingress -n istio-ingress --wait
+    $ kubectl delete namespace istio-ingress
+    {{< /text >}}
+
+### Uninstall the components together
+
+1. Delete the Istio ambient chart:
+
+    {{< tip >}}
+    [By design](https://github.com/helm/community/blob/main/hips/hip-0011.md#deleting-crds),
+    deleting a chart via Helm doesn't delete the installed Custom
+    Resource Definitions (CRDs) installed via the chart.
+    {{< /tip >}}
+
+    {{< text syntax=bash snip_id=delete_ambient_wrapper >}}
+    $ helm delete istio-ambient -n istio-system --wait
+    {{< /text >}}
+
+1. Delete the `istio-system` namespace:
```

Comment:
this isn't done automatically? 🤔

Reply:
It would be, except for the whole "you have to delete CRDs manually" thing, which Helm categorically refuses to do automatically, because kubernetes effectively provides no way to do it safely and unconditionally